test/e2e/lifecycle/bootstrap/util.go

```diff
@@ -74,10 +77,12 @@ func addSecretExpiration(s *v1.Secret, expiration string) {
 	s.Data[bootstrapapi.BootstrapTokenExpirationKey] = []byte(expiration)
 }
 
+// TimeStringFromNow returns the time as a string from now
```

Comment:
Could you include an example of the time string to know the format easily?

Reply:
TimeStringFromNow(600 * time.Second) 2019-12-03T14:30:40+08:00  ->  2019-12-03T14:30:40+08:00 

// TimeStringFromNow returns the time as a string from now
// 2019-12-03T14:30:40+08:00  TimeStringFromNow(600 * time.Second) 
// and then returns 2019-12-03T14:30:40+08:00 
how about this?Thanks

Reply:
just
```
// TimeStringFromNow returns the time as a string from now
// e.g: 2019-12-03T14:30:40+08:00
```
can be enough :-)

Reply:
sure.Thanks